resources/measures/ResidentialHVACCentralSystemFanCoil/measure.rb

```diff
@@ -62,6 +69,9 @@ def run(model, runner, user_arguments)
 
     fan_coil_heating = runner.getBoolArgumentValue("fan_coil_heating", user_arguments)
     central_boiler_fuel_type = HelperMethods.eplus_fuel_map(runner.getStringArgumentValue("central_boiler_fuel_type", user_arguments))
+    if fan_coil_heating
```

Comment:
Is this check needed? It looks like this logic is built into options_lookup, at least for the one `ResidentialHVACCentralSystemFanCoil` option with `fan_coil_heating=false` (`Fan Coil Cooling Only`), the flue gets set to false.

Reply:
My thought was that you could use the fan coil measure for cooling only. Then also have, e.g., the furnace measure for heating only. In such a case, you wouldn't want to overwrite the fan coil's `has_hvac_flue=false` with the furnace's `has_hvac_flue=true`.

Reply:
Makes sense